### DISCUSSION TITLE
Incoherent information between glossary and carbon efficiency key concept for methane

### DISCUSSION THREAD

**Discussion by russelltrow** [Green-Software-Foundation/learn#183] [General]
An LFC131 learner has raised a question on the LF course forum:

> In the glossary, the warming effected of methane is stated to be 25 times that of CO2, however in the key concepts of the Carbon efficiency section, it is quoted to be about 80 times that of CO2. The numbers are either incoherent or used in an imprecise manner. Either way, it would be good to address that discrepancy. Looking it up, it seems that the warming power of methane is about 84 times of CO2 when emitted but falls to 28 times after 100 years (because it dissipates much faster than CO2), so it might be that both numbers are correct, just using a different referential.

https://forum.linuxfoundation.org/discussion/863882/incoherent-information-between-glossary-and-carbon-efficiency-key-concept-for-methane

@greenhsu123 do you have any feedback on the points raised?

**Comment by greenhsu123** [comment]
Hi, Thank you for raising the discrepancy; great spot! You are correct that the warming effect of methane is about 28 times that of carbon dioxide over 100 years, while 84 times over 20 years. [1]. We will go ahead and amend the content. 
[1]- https://energy.ec.europa.eu/topics/oil-gas-and-coal/methane-emissions_en

**Comment by russelltrow** [comment]
I have responded on the LF forum and I will get the LFC131 course content updated accordingly.
